web/concrete/src/Application/Application.php

```diff
@@ -99,8 +101,10 @@ public function clearCaches()
         Core::make('cache')->flush();
         Core::make('cache/expensive')->flush();
```

Comment:
not sure what to replace, shall we replace `new Logger` with this too `$logger = $this['Concrete\Core\Logging\Logger'];`